### PR TITLE
Adjust hotkeys before the release

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

File: src/CanvasView.tsx [hunk 1]
```diff
@@ -127,10 +127,10 @@ export const CanvasView: React.FC = () => {
               variant="secondary"
             />
             <IconButton
-              aria-label="Fit to view"
-              title="Fit to view"
+              aria-label="Fit to content"
+              title="Fit to content"
               icon={<CompressIcon />}
-              onClick={() => canvasService.send('FIT_TO_VIEW')}
+              onClick={() => canvasService.send('FIT_TO_CONTENT')}
```
Comment (farskid) [inline src/CanvasView.tsx:133]: why are we changing this term?

File: src/SettingsPanel.tsx [hunk 2]
```diff
@@ -72,18 +69,10 @@ const KeyboardShortcuts = () => (
...
-              <Kbd>f</Kbd>
-            </VStack>
-          </Td>
-          <Td>Fit to view</Td>
-        </Tr>
-        <Tr>
-          <Td>
-            <VStack alignItems="flex-start">
-              <Kbd>r</Kbd>
+              <Kbd>Shift</Kbd> + <Kbd>1</Kbd>
```
Comment (farskid) [inline src/SettingsPanel.tsx:72]: Remove the `VStack` entirely in favor of:

image

File: src/CanvasView.tsx (hunk 1 above)
Comment (Andarist) [reply src/CanvasView.tsx:133]: https://github.com/statelyai/registry/pull/196#discussion_r747213455

Comment (Andarist) [reply src/CanvasView.tsx:133]: This is the one that The Piano Man prefers

**Review by farskid — CHANGES_REQUESTED** [review]
Pending the change requested in the comment

File: src/SettingsPanel.tsx (hunk 2 above)
Comment (farskid) [reply src/SettingsPanel.tsx:72]: Because with `VStack` the `+` is not shown in the UI

**Review by farskid — APPROVED** [review]

**Pull request merged**